health/api/src/test/scala/com/cerner/beadledom/health/dto/HealthDependenciesDtoSpec.scala

```diff
@@ -0,0 +1,22 @@
+package com.cerner.beadledom.health.dto
+
+import org.scalatest.{FunSpec, MustMatchers}
+
+import scala.collection.JavaConverters._
+
+class HealthDependenciesDtoSpec extends FunSpec with MustMatchers with JacksonSerializationBehaviors {
+
+  val healthDependency = HealthDependencyDto.builder()
+                         .setHealthy(true)
+                         .setId("1")
+                         .setName("name")
+                         .setMessage("message")
+                         .setPrimary(true)
+                         .setLinks(LinksDto.builder().setSelf("link").build())
+                         .build()
+
+  var healthDependenciesDto = HealthDependenciesDto.builder()
+                              .setDependencies(List(healthDependency).asJava).build()
+
+  it must behave like serializableByJackson(healthDependenciesDto)
```

Comment:
Seeing this now in an actual `behave` statement, I wonder if the behavior method would be better named as `aJacksonSerializableObject`

Reply:
I like it, https://github.com/cerner/beadledom/pull/100/commits/ac509b10974ebd5fe955358ff52a3000348ee107